dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxrsMonitor-1.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxrsMonitor-1.0
+Manifest-Version: 1.0
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.monitor-1.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxrs-2.0)(osgi.identity=com.ibm.websphere.appserver.jaxrs-2.1)))"
+IBM-Install-Policy: when-satisfied
+-bundles=com.ibm.ws.jaxrs.2.x.monitor
+kind=ga
```

Comment:
beta first please

Reply:
Hi Tom,   Is it documented somewhere that all features must go into a beta before inclusion?   This feature is only adding metrics and monitoring capability and I wasn't thinking it would need to go out in a beta first.

Reply:
done